python/paddle/trainer_config_helpers/layers.py

```diff
@@ -5876,3 +5878,40 @@ def prelu_layer(input,
         layer_type=LayerType.PRELU,
         parents=input,
         size=l.config.size)
+
+
+@layer_support()
+@wrap_name_default('pixel_softmax')
+def pixel_softmax_layer(input, name=None, layer_attr=None):
+    """
+    This layer calculate softmax in image channel dimension
+
+    The example usage is:
+
+    .. code-block:: python
+
+       prelu = pixel_softmax(input=layer, name='softmax')
+
+    :param name: Name of this layer.
+    :type name: basestring
+    :param input: The input layer.
+    :type input: LayerOutput
```

Comment:
It's better to put input in front of the name to keep the order same with input arguments.